Clients/src/presentation/pages/ComplianceTracker/1.0ComplianceTracker/index.tsx

```
@@ -95,27 +123,55 @@ const ComplianceTracker = () => {
+                />
+              </div>
+            ) : (
               <ControlCategoryTile
                 key={controlCategory.id}
                 controlCategory={controlCategory}
                 onComplianceUpdate={fetchComplianceData}
               />
-            ))}
-      </Stack>
+            )
+          )}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)